openfe/setup/ligand_network_planning.py

```diff
@@ -17,31 +17,46 @@
 
 from lomap import generate_lomap_network, LomapAtomMapper
 from lomap.dbmol import _find_common_core
+from konnektor.network_planners import (
+    StarNetworkGenerator,
+    MaximalNetworkGenerator,
+    RedundantMinimalSpanningTreeNetworkGenerator,
+    MinimalSpanningTreeNetworkGenerator,
+    ExplicitNetworkGenerator,
+)
+from konnektor import network_analysis, network_planners, network_tools
 
 
 def _hasten_lomap(mapper, ligands):
-    """take a mapper and some ligands, put a common core arg into the mapper """
+    """take a mapper and some ligands, put a common core arg into the mapper"""
     if mapper.seed:
         return mapper
 
     try:
-        core = _find_common_core([m.to_rdkit() for m in ligands],
-                                 element_change=mapper.element_change)
+        core = _find_common_core(
+            [m.to_rdkit() for m in ligands],
+            element_change=mapper.element_change,
+        )
     except RuntimeError:  # in case MCS throws a hissy fit
         core = ""
 
     return LomapAtomMapper(
-        time=mapper.time, threed=mapper.threed, max3d=mapper.max3d,
-        element_change=mapper.element_change, seed=core,
-        shift=mapper.shift
+        time=mapper.time,
+        threed=mapper.threed,
+        max3d=mapper.max3d,
+        element_change=mapper.element_change,
+        seed=core,
+        shift=mapper.shift,
     )
 
 
 def generate_radial_network(
     ligands: Iterable[SmallMoleculeComponent],
-    central_ligand: Union[SmallMoleculeComponent, str, int],
     mappers: Union[AtomMapper, Iterable[AtomMapper]],
+    central_ligand: Union[SmallMoleculeComponent, str, int, None],
```

Comment:
I wouldn't change the order unless we really have to - i.e. it's technically API breaking in the case that someone just passes arguments without specifying them?